vtex/utils/openapi/vcs.openapi.gen.ts

```diff
@@ -128,6 +128,14 @@ Href?: string
 DocumentId?: string
 }
 }
+/**
+ * Retrieves user document information.
+ */
+"GET /api/dataentities/CL/search?email=:email&_fields=document": {
```

Comment:
My feeling is that "CL" should be dynamic and not static and may be the query params email and _fields too.

Reply:
> My feeling is that "CL" should be dynamic and not static and may be the query params email and _fields too.

I agree and I uploaded the fix to make this possible. Can you review?